chpl/chpl-api/e2e/script/newman-run.js

```diff
@@ -32,12 +32,25 @@ const apiKeyControllerTests = {
     },
 };
 
-const certifiedProductControllerTests = {
+const certifiedProductControllerDataTests = {
   ...commonOptions,
   collection: collection_path + '/certified-product-controller.postman_collection.json',
+  folder: 'GET : certified_product: data driven tests',
+  iterationData: data_path + '/chplids-test-data.json',
```

Comment:
This capitalization does not match the capitalization of the file. It might work on a windows machine, but won't on a linux one

Reply:
Done